.vsconfig

```diff
@@ -0,0 +1,86 @@
+    "Microsoft.VisualStudio.Component.SQL.SSDT",
+    "Microsoft.VisualStudio.Component.SQL.DataSources",
+    "Component.Microsoft.Web.LibraryManager",
+    "Component.Microsoft.WebTools.BrowserLink.WebLivePreview",
+    "Microsoft.VisualStudio.ComponentGroup.Web",
+    "Microsoft.NetCore.Component.Runtime.6.0",
+    "Microsoft.NetCore.Component.SDK",
+    "Microsoft.VisualStudio.Component.FSharp",
+    "Microsoft.ComponentGroup.ClickOnce.Publish",
+    "Microsoft.NetCore.Component.DevelopmentTools",
+    "Microsoft.VisualStudio.Component.FSharp.WebTemplates",
+    "Microsoft.VisualStudio.Component.DockerTools",
+    "Microsoft.NetCore.Component.Web",
+    "Microsoft.VisualStudio.Component.WebDeploy",
+    "Microsoft.VisualStudio.Component.AppInsights.Tools",
+    "Microsoft.VisualStudio.Component.Web",
+    "Microsoft.Net.Component.4.8.TargetingPack",
+    "Microsoft.Net.ComponentGroup.4.8.DeveloperTools",
+    "Microsoft.VisualStudio.Component.AspNet45",
+    "Microsoft.VisualStudio.Component.AspNet",
+    "Component.Microsoft.VisualStudio.Web.AzureFunctions",
+    "Microsoft.VisualStudio.ComponentGroup.AzureFunctions",
+    "Microsoft.VisualStudio.ComponentGroup.Web.CloudTools",
+    "Microsoft.VisualStudio.Component.DiagnosticTools",
+    "Microsoft.VisualStudio.Component.EntityFramework",
+    "Microsoft.VisualStudio.Component.Debugger.JustInTime",
+    "Component.Microsoft.VisualStudio.LiveShare",
+    "Microsoft.VisualStudio.Component.WslDebugging",
+    "Microsoft.VisualStudio.Component.IntelliCode",
+    "Microsoft.Net.Component.4.6.2.TargetingPack",
+    "Microsoft.Net.Component.4.7.TargetingPack",
+    "Microsoft.VisualStudio.Workload.NetWeb",
+    "Microsoft.VisualStudio.Component.VC.CoreIde",
+    "Microsoft.VisualStudio.Component.VC.Tools.x86.x64",
+    "Microsoft.VisualStudio.Component.Windows10SDK.19041",
+    "Microsoft.VisualStudio.Component.ManagedDesktop.Prerequisites",
+    "Microsoft.ComponentGroup.Blend",
+    "Microsoft.VisualStudio.Component.FSharp.Desktop",
+    "Microsoft.VisualStudio.ComponentGroup.MSIX.Packaging",
+    "Microsoft.VisualStudio.Workload.ManagedDesktop",
+    "Microsoft.VisualStudio.Component.VC.Redist.14.Latest",
+    "Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core",
+    "Microsoft.VisualStudio.Component.VC.Tools.ARM64",
+    "Microsoft.VisualStudio.Component.VC.CLI.Support",
+    "Microsoft.VisualStudio.Component.Windows10SDK.18362",
+    "Microsoft.VisualStudio.Workload.NativeDesktop",
+    "Microsoft.VisualStudio.Component.VC.Tools.ARM",
+    "Microsoft.Net.ComponentGroup.TargetingPacks.Common",
+    "Microsoft.Net.Component.4.6.TargetingPack",
+    "Microsoft.VisualStudio.Component.Git",
+    "Microsoft.VisualStudio.Component.LinqToSql",
+    "Microsoft.NetCore.Component.Runtime.3.1",
+    "Microsoft.NetCore.Component.Runtime.5.0",
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
+    "Microsoft.Net.Component.4.6.2.SDK",
+    "Microsoft.Net.Component.4.7.SDK"
```

Comment:
For the .NET Framework SDKs, we only need `Microsoft.Net.Component.4.8.SDK` as long as we have the other targeting packs (4.6.1, etc).